casestudies/build.sh

```diff
@@ -12,4 +12,4 @@ patch -p0 < ../diffs/mime.diff
 cd ..
 
 # patch hippy-c-cgi
-patch -p0 < diffs/hippycgi.diff
+#patch -p0 < diffs/hippycgi.diff
```

Comment:
Okay, seems this needs to stay, as when building pyhyp, the `hippy-c` is moved to `pyhyp`, see https://github.com/softdevteam/pyhyp_experiments/blob/sqmail_update/build.sh#L320